tests/Agent/IntegrationTests/UnboundedIntegrationTests/UnboundedIntegrationTests.csproj

```diff
@@ -51,10 +51,10 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="CouchbaseNetClient" Version="2.3.8" />
     <PackageReference Include="Oracle.ManagedDataAccess" Version="12.1.2400" />
-    <PackageReference Include="xunit" version="2.8.0" />
-    <PackageReference Include="xunit.assert" Version="2.8.0" />
-    <PackageReference Include="xunit.core" Version="2.8.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.0">
+    <PackageReference Include="xunit" version="2.9.0" />
```

Comment:
Is the xunit version bump something that was triggered by another update in this PR, or just a "while we've got the hood up, we might as well" thing?

Reply:
I think there are still some misconfigurations in `dependabot.yml` - my previous dependabot update PRs should have allowed it to update these in https://github.com/newrelic/newrelic-dotnet-agent/pull/2681 but didn't. I'll look into it a bit closer.